lib/msal-common/src/unifiedCache/entities/AccountEntity.ts

```diff
@@ -25,17 +25,31 @@ export class AccountEntity {
     lastModificationTime?: string;
     lastModificationApp?: string;
 
+    /**
+     * Generate Account Id key component as per the schema: <home_account_id>-<environment>
+     */
+    generateAccountId(): string {
+        const accountId: Array<string> = [this.homeAccountId, this.environment];
+        return accountId.join(Separators.CACHE_KEY_SEPARATOR).toLowerCase();
+    }
+
+    /**
+     * Generate Account Id key component as per the schema: <home_account_id>-<environment>
+     */
+    generateRealm(): string {
+        return (this.realm || "").toLowerCase();
```

Comment:
I think empty space is used in multiple places, consider adding it to the constants file

Reply:
+1 for EMPTY_STRING constant